projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java

```diff
@@ -109,7 +109,8 @@ public String startDataJobExecution(
       // We default to 'manual'; this also served for the purposes of backwards compatibility
       annotations.put(
           JobAnnotation.EXECUTION_TYPE.getValue(),
-          jobExecutionRequest.getStartedBy().contains("scheduled")
```

Comment:
Do we have a clue why the value is null, and not a username or some identifier? We hit this in an internal deployment of the Control Service, and it was unexpected to say the least.

Reply:
The frontend doesn't send a startedBy. 
Is this not expected? If it is not expected we need to change the open api definition.

Reply:
This code was changed recently. 
This caused the regression.

Reply:
Oopsie, yup, I saw the change. Should've `git blame`'ed the file first. Sorry